snapcraft/storeapi/_upload.py

```diff
@@ -48,7 +54,8 @@ def upload(binary_filename, metadata_filename='', metadata=None, config=None):
         return
     name = match.groupdict()['name']
 
-    logger.info('Uploading files...')
+    logger.info('')
```

Comment:
This is only here to print a newline before the progress bar (it can't be contained within the progress bar or it keeps printing newlines). I just thought it looked a little nicer with some spacing.